docs/spec/spans/rum_v3_span.json

```diff
@@ -175,39 +182,35 @@
                                 }
                             }
                         },
-                        "message": {
+                        "mg": {
                             "$ref": "../message.json"
                         }
                     }
                 },
-                "duration": {
+                "d": {
                     "type": "number",
                     "description": "Duration of the span in milliseconds"
                 },
-                "name": {
+                "n": {
                     "type": "string",
                     "description": "Generic designation of a span in the scope of a transaction",
                     "maxLength": 1024
                 },
-                "stacktrace": {
+                "st": {
                     "type": ["array", "null"],
                     "description": "List of stack frames with variable attributes (eg: lineno, filename, etc)",
                     "items": {
-                        "$ref": "../stacktrace_frame.json"
+                        "$ref": "../rum_v3_stacktrace_frame.json"
                     },
                     "minItems": 0
                 },
-                "sync": {
+                "sy": {
                     "type": ["boolean", "null"],
                     "description": "Indicates whether the span was executed synchronously or asynchronously."
                 }
             },
-            "required": ["duration", "name", "type", "id","trace_id", "parent_id"]
+            "required": ["d", "n", "t", "id","trace_id", "parent_id"]
```

Comment:
`trace_id` and `parent_id` must be optional right?

Reply:
yeah, that is job for another pr :)